models/user.go

```diff
@@ -0,0 +1,29 @@
+package models
+
+import (
+	"github.com/beego/beego/v2/adapter/orm"
+)
+
+type User struct {
+	Base
+
+	Name           string
+	Email          string `orm:"unique"`
+	HashedPassword string
+}
+
+func CreateUser(u *User) (id int64, err error) {
+	orm := orm.NewOrm()
+	return orm.Insert(u)
+}
+
+func (u *User) IsExist() bool {
```

Comment:
Just noticed this issue but `IsExist` is not proper English 🙈 What would be valid: `Exist`, `IsExistingUser`